test/filter-dropdown/filter-dropdown-category.test.js

```diff
@@ -38,14 +37,18 @@ describe('d2l-labs-filter-dropdown-category', () => {
 	it('instantiating the element works', () => {
 		expect('d2l-labs-filter-dropdown-category').to.equal(categories[0].tagName.toLowerCase());
 	});
-	it('attributes are set correctly (including d2l-tab-panel attributes)', () => {
+	it('attributes are set correctly (including d2l-tab-panel attributes)', async() => {
+		await categories[0].updateComplete;
+		await categories[1].updateComplete;
```

Comment:
A nicer solution would have been to move this up into `beforeEach`, but at least one of the tests ('changing the category tab triggers the d2l-labs-filter-dropdown-category-selected event') immediately `await`s an event that's already fired if we await `updateComplete`. These probably need a rethink at some point.